evap/staff/tests/test_views.py

```diff
@@ -1929,6 +1930,32 @@ def setUpTestData(cls):
         cls.answer = 'should show up'
         baker.make(TextAnswer, contribution=contribution, question=question, answer=cls.answer)
 
+        cls.evaluation2 = baker.make(
+            Evaluation,
+            pk=2,
+            course=baker.make(Course, semester=semester),
+            participants=[student1],
+            voters=[student1, cls.student2],
+            vote_start_datetime=datetime.datetime.now() - datetime.timedelta(days=5),
+            vote_end_date=datetime.date.today() - datetime.timedelta(days=4),
+            can_publish_text_results=True
+        )
+
+        cls.evaluation2.general_contribution.questionnaires.set([top_general_questionnaire])
```

Comment:
Do we need this here? Do we need this above (with the LIKERT question)? I don't see why it would be necessary, but I'm probably just missing something.

Reply:
no, we don't need it here.